lib/internal/errors.js

```diff
@@ -139,6 +142,7 @@ E('ERR_IPC_DISCONNECTED', 'IPC channel is already disconnected');
 E('ERR_IPC_ONE_PIPE', 'Child process can have only one IPC pipe');
 E('ERR_IPC_SYNC_FORK', 'IPC cannot be used with synchronous forks');
 E('ERR_MISSING_ARGS', missingArgs);
+E('ERR_MULTIPLE_WRITE_CALLBACK', 'write callback called multiple times');
```

Comment:
If I recall correctly, there are a couple of places in core with a similar error. A more generic error message may be appropriate.